cypress-tests/cypress/support/pageBuilder/pbCreatePageTemplate.ts

```diff
@@ -4,7 +4,7 @@ declare global {
     // eslint-disable-next-line @typescript-eslint/no-namespace
     namespace Cypress {
         interface Chainable {
-            pbCreatePageTemplate(data: any): Promise<any>; // Update the data type as needed
+            createPageTemplate(data: any): Promise<any>; // Update the data type as needed
```

Comment:
Should be in sync with L43, so `pbCreatePageTemplate`.

Reply:
Instead of `Promise<any>`, let's maybe do:

```ts
Promise<{id : string, title: string, slug: string, tags: string[], ... }>
```

I know we said no to bother with this much at this point, so if you want, we can tackle this in a separate PR. But still, maybe we can tackle this along the way immediately.

Reply:
Let's check the `CreatePageTemplate` mutation below and remove unneeded fields in the data object. We probably just need the first three or so.